librz/il/il_routines.c

```diff
@@ -0,0 +1,125 @@
+// SPDX-FileCopyrightText: 2023 Rot127 <unisono@quyllur.org>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_il/rz_il_opcodes.h>
+#include <rz_types.h>
+
```

Comment:
Add `\file` describing the purpose of these functions (helpers to form bitwise operations)